chromium_src/chrome/common/pref_names.h

```diff
@@ -0,0 +1,21 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_CHROME_COMMON_PREF_NAMES_H_
+#define BRAVE_CHROMIUM_SRC_CHROME_COMMON_PREF_NAMES_H_
+
+#include "build/build_config.h"
+
+namespace prefs {
+// Used by BraveVpnDnsObserverService to save cloudflare server url when
```

Comment:
save -> set

Reply:
done in https://github.com/brave/brave-core/pull/15989/commits/c2b399158c4310c6ed1d1d94890a2a0c179f9721